GlobalStaticVersion.props

```diff
@@ -8,12 +8,12 @@
     <SemanticVersionMajor>2</SemanticVersionMajor>
     <SemanticVersionMinor>2</SemanticVersionMinor>
     <SemanticVersionPatch>0</SemanticVersionPatch>
-    <PreReleaseMilestone>beta6</PreReleaseMilestone>
+    <PreReleaseMilestone></PreReleaseMilestone>
```

Comment:
you do not need this change. Just ruin "stable" build and it will not append the postfix